cluster/gce/util.sh

```diff
@@ -2332,7 +2342,8 @@ function check-existing() {
 }
 
 function check-network-mode() {
-  local mode="$(gcloud compute networks list --filter="name=('${NETWORK}')" --project ${NETWORK_PROJECT} --format='value(x_gcloud_subnet_mode)' || true)"
+  local mode
+  mode="$(gcloud compute networks list --filter="name=('${NETWORK}')" --project ${NETWORK_PROJECT} --format='value(x_gcloud_subnet_mode)' || true)"
```

Comment:
Superfluous quotes for this type of assignment.

$ a=$(echo foo bar)           
$ echo $a
foo bar
$